packages/lesswrong/components/admin/AdminHome.tsx

```diff
@@ -66,6 +72,8 @@ const AdminHome = ({ classes }: {
         <li><Link className={classes.link} to="/reviewAdmin">Review Admin (current year)</Link></li>
         <li><Link className={classes.link} to="/admin/migrations">Migrations</Link></li>
         <li><Link className={classes.link} to="/admin/synonyms">Search Synonyms</Link></li>
+        <li><span className={classes.link} onClick={refreshDbSettings}>Refresh DB Settings</span></li>
```

Comment:
You could _imagine_ adding a disabled class, and disabling if refreshing. (But you know, admin page and all that.)